<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataSelectDialog</class>
 <widget class="QDialog" name="DataSelectDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RIVET: Select Data</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="rawDataFileButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Raw Data File</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="computeButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>380</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Compute</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>400</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>You can start from a point cloud or a bifiltration file.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="persistenceFileButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Open RIVET File</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>400</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Re-load a saved computation.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="shortcutButton1">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>140</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Use Sample3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="shortcutButton2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>170</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Use CircleData4</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>378</width>
     <height>136</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="xlabelLabel">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>66</y>
      <width>50</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>x-label:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ylabelLabel">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>99</y>
      <width>49</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>y-label:</string>
    </property>
   </widget>
   <widget class="QLabel" name="xbinLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>66</y>
      <width>44</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>x-bins:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ybinLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>99</y>
      <width>44</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>y-bins:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="xbinSpinBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>66</y>
      <width>57</width>
      <height>27</height>
     </rect>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="ybinSpinBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>99</y>
      <width>57</width>
      <height>27</height>
     </rect>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
   </widget>
   <widget class="QLabel" name="homDimLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>33</y>
      <width>150</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Homology Dimension:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="homDimSpinBox">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>33</y>
      <width>48</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>83</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Parameters</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xlabelBox">
    <property name="geometry">
     <rect>
      <x>222</x>
      <y>66</y>
      <width>146</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>time</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ylabelBox">
    <property name="geometry">
     <rect>
      <x>222</x>
      <y>99</y>
      <width>146</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>distance</string>
    </property>
   </widget>
  </widget>
  <zorder>frame</zorder>
  <zorder>rawDataFileButton</zorder>
  <zorder>computeButton</zorder>
  <zorder>label</zorder>
  <zorder>persistenceFileButton</zorder>
  <zorder>label_2</zorder>
  <zorder>shortcutButton1</zorder>
  <zorder>shortcutButton2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
